jdaviz/configs/cubeviz/plugins/tests/test_parsers.py

```diff
@@ -224,7 +242,7 @@ def test_manga_cube(cubeviz_helper):
     se.function = "Mean"
     se.extract()
     extracted_max = cubeviz_helper.get_data("Spectrum (mean)").max()
-    assert_allclose(extracted_max.value, 2.836957E-18)
+    assert_allclose(extracted_max.value, 5.566169e-18)
```

Comment:
That is quite a jump. Does this mean MANGA result has been wrong all this time? 😱

Reply:
Yes, I think that `mean` extract has been wrong (well, wrong if you don't want to average the masked values into your spectrum...) at least since the spectral extraction was changed.